frontend/cypress/perf/Graphload.spec.ts

```diff
@@ -0,0 +1,74 @@
+describe('graphload', () => {
+
+    http://localhost:3000/console/graph/namespaces/
+        // ?traffic=grpc%2CgrpcRequest%2Chttp%2ChttpRequest%2Ctcp%2CtcpSent
+        // &graphType=versionedApp
+        // &namespaces=bookinfo%2Ccert-manager
+        // &duration=60
+        // &refresh=15000
+        // &layout=kiali-dagre
+        // &namespaceLayout=kiali-dagre
```

Comment:
nitpick: Can this be cleaned up? Not sure what it's documenting.

Reply:
Cleaned!